src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,15 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        text = text.toLowerCase();
+        StringBuilder charactersAndDigitsSB = new StringBuilder();
+        for (int i = 0; i < text.length(); i++) {
```

Comment:
what about checklist rules? 
https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_03.html

Reply:
Reviewed and done.